docs/package_templates/header_only/all/test_package/CMakeLists.txt

```diff
@@ -1,11 +1,8 @@
 cmake_minimum_required(VERSION 3.15)
 project(test_package LANGUAGES C) # if the project is pure C
-# project(test_package LANGUAGES CXX) # if the project uses c++
+# project(test_package LANGUAGES CXX) # if the project uses C++
```

Comment:
I suggest dropping these comments. It's a very basic CMake feature and should be fairly obvious.

And I dislike having to clean these every time I copy the template, but that's just me. :slightly_smiling_face: